src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -10,13 +11,18 @@
  */
 public class ClearCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear";
+    public static final String COMMAND_WORD = "clear-confirm";
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
-
+    public static final String MESSAGE_CONFIRMATION = "You are attempting to clear all addresses in the memory."
+            + " Add '-confirm' to clear all addresses.";
```

Comment:
How about changing it to 
```Add '-confirm' to the end of the command```
 to make the instruction less ambiguous?

Reply:
Hmm good point. I will edit it. Thanks.